### PR TITLE
10.0 backport sequence category

### PR DESCRIPTION
**Pull request by ovnicraft**

### CODE REVIEW

**Review by moylop260 — COMMENTED** [review]
Could you check lint putput please?
======= Testing test_flake8 ========
product_sequence/models/product_product.py:17:80: E501 line too long (85 > 79 characters)
product_sequence/models/product_product.py:50:80: E501 line too long (80 > 79 characters)
product_sequence/models/product_category.py:36:80: E501 line too long (80 > 79 characters)
product_sequence/models/product_category.py:39:80: E501 line too long (82 > 79 characters)
product_sequence/models/product_category.py:41:80: E501 line too long (82 > 79 characters)

**Comment by ovnicraft** [comment]
@moylop260 2 will not solved.

**Comment by ovnicraft** [comment]
@pedrobaeza can you review it please

File: product_sequence/models/product_product.py [hunk 1]
```diff
@@ -1,47 +1,70 @@
...
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import models, fields, api, _
 
 
 class ProductProduct(models.Model):
-    _inherit = 'product.product'
+    _inherit = "product.product"
```
Comment (moylop260) [inline product_sequence/models/product_product.py:12]: Hi @ovnicraft 
Here we have the same situation about another PR

**Pull request closed without merging**